book/creating-a-basic-website/about-the-sulu-content-architecture.rst

```diff
@@ -1,6 +1,11 @@
 About the Sulu Content Architecture
 ===================================
 
+We already heard something about :doc:`../introduction/content-architecture`
+in the introduction. 
+Now we are starting to code will dig a little bit deeper.
+
+
```

Comment:
yes. i want to cross reference and we said we wont to have an introduction and short summary or outview on every page.